src/pages/extending-chat-widget/chat-widget-adapters/angular-adapter/index.mdx

```diff
@@ -0,0 +1,312 @@
+---
+weight: 40
+category: "extending-chat-widget"
+subcategory: "chat-widget-adapters"
+title: "Angular adapter"
+tagline: ""
+desc: "This library allows to render and interact with the LiveChat Chat Widget inside an Angular application."
+---
+
+This library allows to render and interact with the LiveChat Chat Widget inside an [Angular](http://angular.io/) application.
+
+## Installation
+
+### Using npm
+
+```bash
+npm install @livechat/widget-angular
+```
+
+### Using a script tag
+
+```js
+<script src="http://unpkg.com/@livechat/widget-core@1.0.1"></script>
+<script src="https://unpkg.com/@livechat/widget-angular@1.0.1"></script>
+```
+
+## Usage
+
+### Render
+
+```js
+// app.module.ts
+
+import { NgModule } from "@angular/core";
+import { LiveChatWidgetModule } from "@livechat/widget-angular";
+
+@NgModule({
+  /* ... */
+  imports: [LiveChatWidgetModule],
+})
+export class AppModule {}
+```
+
+```js
```

Comment:
Perhaps it's worth using the CodeBlock component as it comes with the copy-to-clipboard feature? wdyt?

Reply:
I had some troubles using `Code` and `CodeSample` components as the `import` statements in code examples are interfering with those components causing build error 🙈

Reply:
Oh 😅